java/ql/src/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -93,6 +98,81 @@ private module Cached {
 
 import Cached
 
+private module StoreTaintSteps {
```

Comment:
Document this for the benefit of external readers who ought to know this is a stop-gap / temporary measure (and what it intends to do).